erpnext/stock/doctype/purchase_receipt/purchase_receipt.py

```diff
@@ -230,17 +231,28 @@ def get_gl_entries(self, warehouse_account=None):
 						"debit": stock_value_diff
 					}, warehouse_account[d.warehouse]["account_currency"], item=d))
 
+					if d.from_warehouse:
+						gl_entries.append(self.get_gl_dict({
+							"account":  warehouse_account.get(d.from_warehouse, {}).get('account') \
```

Comment:
Why `or stock_rbnb `?

Reply:
Also in line 228, `against` should be `warehouse_account.get(d.from_warehouse, {}).get('account') if d.from_warehouse else stock_rbnb `